src/boot/bin/XMLToSARIF.sh

```diff
@@ -1,11 +1,12 @@
 #!/bin/bash
 print_usage() {
-    echo "Usage: XMLToSARIF.sh -i <inputXmlReport> [-o <outputSarifReport>] [-t <toolOrJavaHomeDir>]"
+    echo "Usage: XMLToSARIF.sh -i <inputXmlReport> [-o <outputSarifReport>] [-t <toolOrJavaHomeDir>] [-p <projectPaths>]"
```

Comment:
Can we call the option name `projectRootPaths`? If so, please change other veriable names and descriptions.